Code/Tools/ProjectManager/Source/PythonBindings.h

```diff
@@ -44,6 +44,7 @@ namespace O3DE::ProjectManager
         bool ValidateRepository(const QString& repoUri) override;
 
         // Gem
+        AZ::Outcome<GemInfo> CreateGem(const QString& templatePath, const GemInfo& gemInfo, bool registerGem = true) override;
```

Comment:
While you are in here can you mark the `m_lock` recursive_mutex `mutable` and add `const` to the `ExecuteWithLock` signature.

Reply:
Ah good idea!